tests/Integration/ResourceStatusSystem/Test_ResourceManagement.py

```diff
@@ -27,15 +30,15 @@ def test_addAndRemove():
   # ...............................................................................
 
   res = rsClient.addOrModifyAccountingCache('TestName12345', 'plotType', 'plotName', 'result', dateEffective, lastCheckTime)
-  assert res['OK'] == True
+  assert res['OK'] is True
```

Comment:
I got where it is coming from. In fact we are setting the PYTHONOPTIMIZE env variable in the bashrc, this is the culprit but also handy: 

    $ which python
    /cvmfs/dirac.egi.eu/dirac/pro/Linux_x86_64_glibc-2.12/bin/python
    [atsareg:/cvmfs] $ unset PYTHONOPTIMIZE
    [atsareg:/cvmfs] $ python
    Python 2.7.12 (default, Dec 17 2016, 21:07:48)
    [GCC 4.4.7 20120313 (Red Hat 4.4.7-17)] on linux2
    Type "help", "copyright", "credits" or "license" for more information.
    >>> print __debug__
    True
    >>> assert False
    Traceback (most recent call last):
      File "<stdin>", line 1, in <module>
    AssertionError

  So, this is not hard compiled. I think we should leave it like that and unset the PYTHONOPTIMIZE in the testing environment